dashboards-reports/public/components/report_definitions/delivery/delivery.tsx

```diff
@@ -84,6 +85,14 @@ export function ReportDelivery(props: ReportDeliveryProps) {
   const handleSendNotification = (e: { target: { checked: boolean }; }) => {
     setSendNotification(e.target.checked);
     includeDelivery = e.target.checked;
+    if (includeDelivery) {
+      reportDefinitionRequest.delivery.title = 'New report';
```

Comment:
I didn't get this, why `titile` and `textDescription` are set with hard value. Shouldn't those read from user input?

Reply:
These are default placeholder values, if the user wants to change them they can.